recipes-tag/hdf5-lz4/meta.yaml

```diff
@@ -11,9 +11,9 @@ build:
 
 requirements:
     build:
-        - hdf5 1.10.2
+        - hdf5 =1.10.2
```

Comment:
Should it be a single `=` or double `==`?

Reply:
https://docs.conda.io/projects/conda-build/en/master/source/package-spec.html#build-version-spec does not have any single `=` as a version matching.

Reply:
I didn't read the description carefully. I see now that it's the same as in https://github.com/NSLS-II/lightsource2-recipes/blob/c7380dd7aa6f7393581e3a262c417e177feb50ef/recipes-tag/analysis/meta.yaml#L28